### PR TITLE
Removed dayBars from Mobile Menu

### PR DESCRIPTION
**Pull request by ORiveraJr84**
# Description

As suggested by @bporch84, I have removed the "dayBars" from the menu when viewed on a mobile device via a media query as it may confuse potential users as listed in [this Trello card](https://trello.com/c/V0SQwypv/769-remove-daybars-from-mobile-menu). As further suggested by @bporch84, I will create a future PR that will "light up" the written days in the brand green color as that seems like a better approach that sends a clearer message to the potential users.

dayBars

### CODE REVIEW

**Comment by ORiveraJr84** [comment]
[Remove "dayBars" from mobile menu](https://trello.com/c/V0SQwypv/769-remove-daybars-from-mobile-menu)

File: src/styles/less/Hud.less [hunk 1]
```diff
@@ -1,272 +1,273 @@
...
+  }
 }
 
 .dayBars {
-	display: flex;
-	margin-top: 1%;
+  display: flex;
+  margin-top: 1%;
+
+  @media @mobileLarge {
```
Comment (ghost) [inline src/styles/less/Hud.less]: Please change this to @media screen and (max-width: 500px) to match the other media query in the file so both changes happen at the same time.

**Review by ghost — CHANGES_REQUESTED** [review]
You did a good job with the media query, but please make the change I've suggested so that it works in tandem with the other media query in the file.

File: src/styles/less/Hud.less (hunk 1 above)
Comment (ORiveraJr84) [reply src/styles/less/Hud.less]: Done.

**Review by ORiveraJr84 — COMMENTED** [review]
> You did a good job with the media query, but please make the change I've suggested so that it works in tandem with the other media query in the file.

Done.

**Review by ghost — DISMISSED** [review]

**Review by PVigar88 — DISMISSED** [review]
I think this will clean up the interface nicely.  especially with the highlighting of the day itself

**Review by ashtilawat — DISMISSED** [review]

**Review by PVigar88 — APPROVED** [review]

**Review by Fry-Fr — COMMENTED** [review]
This is interesting... Is the idea to simplify the UI on mobile? This way it is not cluttered.

**Pull request closed without merging**